test/common/libevent_reactor.cpp

```diff
@@ -49,13 +49,19 @@ TEST_CASE("libevent_init_once") {
 
 extern "C" {
 
-static event_base *event_base_new_fail() { return nullptr; }
+#ifndef _MSC_VER
+#define STATIC static
+#else
+#define STATIC /* Nothing */
+#endif
```

Comment:
Required to allow MSVC to find the moked function. (It seems in general MSVC works much better when we provide it with a mocked interface rather than just a mocked function.)